src/main/modules/csrf/index.ts

```diff
@@ -9,25 +8,29 @@ import {
   CSRF_TOKEN_ERROR_URL,
 } from '../../steps/urls';
 
-const { Logger } = require('@hmcts/nodejs-logging');
-const logger: LoggerInstance = Logger.getLogger('app');
-
 export class CSRFToken {
-  public enableFor(app: Application): void {
-    app.use(csurf(), (req, res, next) => {
-      res.locals.csrfToken = req.csrfToken();
-      next();
-    });
+  private C100CSRFRoutes = [
```

Comment:
@hmcts-abduldaim  This is just tactical workaround we added. 
We need to fix this with permanent solution to use file upload or multer(pvt law using).
Could you please investigate how can we fix that & remove these exclusions ?